src/components/BorrowerProfile/DetailsTabs.vue

```diff
@@ -50,6 +50,7 @@
 						@show-definition="showDefinition"
 					/>
 					<repayment-schedule
+						v-if="loan.anonymizationLevel !== 'full'"
```

Comment:
Based on the legacy code I think there is one other check that needs to happen at this level.
- Create computed property that returns true/false if the disbursalDate is past now
- Combine that condition with loan type so we can omit the repayment schedule for Direct loans if the disbursalDate hasn't occurred...

This could be another ticket.

Reply:
@mcstover
After looking through the code more last night and this morning, I don't think that check is actually correct. It's my understanding that a directLoan without a disbursal date has not been disbursed. The loan fundraises to completion then it transitions to funded. At that point, the loan takes a few days to process until it's disbursed, at that point the loan gets a disbursal date. 
If a directLoan doesn't have a disbursal date then we show the following portion of code, 
`<div
				v-if="!isPartnerLoan && this.status === 'fundraising'"
				class="tw-prose"
			>
				<p>
					This loan is for {{ loanAmountFormatted }}.
				</p>
				<p>
					<!-- eslint-disable-next-line max-len -->
					Repayments on this loan will be made in monthly installments of {{ calculateMonthlyPayment }} USD over a period of {{ lenderRepaymentTerm }} months. After the funds are disbursed, the borrower(s) will have the standard 1 month before the first payment is due, and may have an additional grace period per the terms outlined in their loan agreement. The detailed repayment schedule will be published here at the time that the funds for this loan are disbursed.<br>
				</p>
				<p>
					<!-- eslint-disable-next-line max-len -->
					Disbursement and repayments will be made via PayPal, a web-based payment system. Repayments made on delinquent loans will be applied toward the oldest payment due until the loan becomes current.
				</p>
			</div>`
So if I put this disbursal date check to hide the repayment schedule the code above would never be shown.